### PR TITLE
added Lib.print and Lib.println (fixed #5593)

### LINKED ISSUE #5593: [java] no Lib.print
I was experimenting with Sqlite (not really important) but ran into a inconsistency:

```
#if php
import php.Lib;
#elseif neko
import neko.Lib;
#elseif python
import python.Lib;
#elseif cpp
import cpp.Lib;
#elseif java
import java.Lib;
#end

Lib.print('test');
```

export using java:

```
src/Main.hx:xx: characters 2-11 : Class<java.Lib> has no field print
```

### PR DESCRIPTION
**Pull request by MatthijsKamstra**
added Lib.print and Lib.println (fixed #5593)

### CODE REVIEW

File: std/java/Lib.hx [hunk 1]
```diff
@@ -30,6 +30,20 @@ package java;
 {
 
 	/**
+		Print the specified value on the default output.
+	**/
+	public static function print( v : Dynamic ) : Void {
+		untyped __java__("java.lang.System.out.print(v)");
```
Comment (nadako) [inline std/java/Lib.hx]: ouch, does it really have to be `untyped __java__`? i'm pretty sure you can access this api properly in haxe, and then you can also inline those functions.

**Comment by MatthijsKamstra** [comment]
@nadako should this be changed as well?

https://github.com/HaxeFoundation/haxe/blob/development/std/haxe/Log.hx#L97

`Sys.println(str);`

**Comment by nadako** [comment]
Looks good, but I'll leave the review to @waneck :)

**Comment by ncannasse** [comment]
I think we should go the other way for Haxe 4 and remove the print/println functions from platform-specific "Lib" and only keep the Sys ones

**Pull request merged**